PPL/costAnalisis.md

```diff
@@ -0,0 +1,1724 @@
+   <td><strong>7.400€</strong>
+   </td>
+  </tr>
+</table>
+
+
+Teniendo esto en cuenta, durante 37 semanas, que junto con las 15 semanas de desarrollo completamos el 1.er año de vida del producto, amortizamos **7.400€.**
+
+<br/>
+
+## 2.7 Desglose total de los costes de producción
+
+Como aclaración, el coste de periodo de mantenimiento es el salario **BRUTO**, no neto.
+
+
+<table>
+  <tr>
+   <td><strong>Descripción</strong>
+   </td>
+   <td><strong>Duración</strong>
+<p>
+<strong>(Semanas)</strong>
+   </td>
+   <td><strong>Coste</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Periodo de Mantenimiento
+<p>
+(Personal)
+   </td>
+   <td>37
+   </td>
+   <td>33.037,50€
+   </td>
+  </tr>
+  <tr>
+   <td>Costes derivados del mantenimiento (internet y luz)
+   </td>
+   <td>37
+   </td>
+   <td>3.330€
+   </td>
+  </tr>
+  <tr>
+   <td>Costes de licencias
+   </td>
+   <td>37
+   </td>
+   <td>4.114,28€
+   </td>
+  </tr>
+  <tr>
+   <td>Costes de márketing
+   </td>
+   <td>37
+   </td>
+   <td>3693,6€
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td>37
+   </td>
+   <td><strong>44.175,38€</strong>
+   </td>
+  </tr>
+</table>
+
+
+<br/>
+
+# 3. Coste posterior de la ONG
+
+Podemos hacer referencia a dos costes seguros que va a tener la ONG posteriores a nuestra finalización del proyecto, estos son el coste del dominio y el coste del despliegue.
+
+El coste de dominio lo sacamos de la página de IONOS, compramos el dominio .es y en el caso del coste de despliegue utilizando el despliegue F2 de app engine, que son 0.10 euros la hora por el coste del despliegue de la aplicación.
+
+
+<table>
+  <tr>
+   <td colspan="2" ><strong>Costes posteriores de la ONG (por año)</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Coste del dominio
+   </td>
+   <td><strong>50€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Coste del despliegue
+   </td>
+   <td><strong>876€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td><strong>926€</strong>
+   </td>
+  </tr>
+</table>
+
+
+<br/>
+
+# 4. TCO
+
+Para el correcto análisis del TCO tendremos en cuenta los costes asociados al desarrollo y los costes de la puesta en producción de la aplicación durante  los dos primeros años de vida (15 semanas de desarrollo + **89 semanas**).
+
+Para ello calculamos los costes salariales del equipo encargado del mantenimiento y actualización de la aplicación una vez puesta en producción, se contará con un equipo de 5 desarrolladores junior y un senior con una jornada laboral de 10 horas semanales.
+
+También tendremos en cuenta las compensaciones por trabajo en remoto de los trabajadores y los costes de las licencias requeridas.
+
+Por último, se ha analizado y tenido en cuenta para el TCO, los costes de marketing y publicidad, utilizando el mismo personal externo contratado durante el periodo de desarrollo, con las mismas condiciones laborales, es decir, con un contrato de 10h semanales, y con la misma retribución.
+
+
+<table>
```

Comment:
with the previous values, update this table as well

Reply:
same as before, is included in the personal and its updated!